docs/README.md

```diff
@@ -0,0 +1,127 @@
+# Example documentation
+
+This document describes the different software components
+used by the MoveIt 2, ROS 2 and Ignition Gazebo demo,
+where to find them and how they are being called.
+
+This example and documentation uses the name of ***"Ignition Gazebo"***
```

Comment:
I think we can mention here the renaming but we should use Gazebo

Reply:
https://github.com/j-rivero/ign_moveit2_examples/pull/1#issuecomment-1350082938